CalibPPS/TimingCalibration/plugins/PPSTimingCalibrationPCLHarvester.cc

```diff
@@ -42,6 +42,10 @@ class PPSTimingCalibrationPCLHarvester : public DQMEDHarvester {
   const std::string dqmDir_;
   const std::string formula_;
   const unsigned int min_entries_;
+  const double threshold_fraction_of_max_;
+  static constexpr double upper_limit_max_search_ = 20;
+  static constexpr double upper_limit_range_search_ = 20;
```

Comment:
This was 18 before the last update: please check and either confirm or fix

Reply:
Change was intentional, after discussing it internally we decided to switch it to 20.